### PR TITLE
Round done and total attributes added! They are recomputed whenever we sort a round.

### LINKED ISSUE #81: Add expected number of solves and completed number of solves to the Round schema
This will let us reimplement getCompetitorsDoneAndTotal() and getLastRoundResultsCount() in editCompetition.js without hitting unacceptable performance problems.

### PR DESCRIPTION
**Pull request by jfly**
This fixes #81.

Ok, take 2! Live at http://live.clark.jflei.com/manage/StarlightOpen2014/events.

### CODE REVIEW

File: both/methods.js [hunk 1]
```diff
@@ -482,10 +543,71 @@ var RoundSorter = {
...
+      result.solves.forEach(function(solve) {
+        if(solve) {
+          done++;
+        }
+      });
+
+      Results.update(result._id, { $set: { position: position } });
     });
+
+    // Normalize done and total to the number of competitors (results)
```
Comment (FatBoyXPC) [inline both/methods.js]: This should be "participant".

File: both/methods.js [hunk 2]
```diff
@@ -482,10 +543,71 @@ var RoundSorter = {
...
+    var progress = {
+      done: done,
+      total: total,
+    };
+    log.l2(roundId, " setting progress to ", progress);
+    Rounds.update(roundId, { $set: { progress: progress } });
   },
 };
+/*<<<
+function getCompetitorsDoneAndTotal(roundId) {
```
Comment (FatBoyXPC) [inline both/methods.js]: This should be "participant" as well.

Comment (jfly) [inline both/methods.js]: Good catch, fixed!

**Comment by jfly** [comment]
Thanks @FatBoyXPC! I've cleaned up and committed. The 3 pages with icons:

http://live.clark.jflei.com/manage/USNationals2014/events
http://live.clark.jflei.com/manage/USNationals2014/check-in
http://live.clark.jflei.com/manage/USNationals2014/data-entry/333bf/2

**Comment by FatBoyXPC** [comment]
Didn't see any other obvious errors, :shipit:!

**Comment by jfly** [comment]
:radio:!

**Pull request merged**